### PR TITLE
Added test, lint, format, and build

### PR DESCRIPTION
**Pull request by calvinb**
Added a build script to transpile src into lib. I think this is the first time we have done this with one of our libraries, so please give it a good look to see if this looks like the best way to set it up.

### CODE REVIEW

**Comment by calvinb** [comment]
@Prestaul I took a crack at that. Is there a way to test the publish without actually publishing?

**Comment by calvinb** [comment]
kenneth

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"presets": [ "node5" ]
```
Comment (Prestaul) [inline .babelrc]: I think we want to use the node6 presets

File: package.json [hunk 2]
```diff
@@ -2,9 +2,18 @@
...
+  "main": "lib/index.js",
+  "files": [ "lib" ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "babel src -d lib",
+    "format": "jscs --fix ./",
+    "lint": "eslint --fix ./",
+    "pretest": "npm run format && npm run lint",
+    "test": "mocha --require spec/babelhook spec/*.spec.js",
+    "test:watch": "npm test -- -w",
```
Comment (Prestaul) [inline package.json]: Add `test:only` that looks like the current `test` and update `test` to just call `cover`?

**Pull request merged**